### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by GBowman1**
## Describe Changes Made
We setup the Database schema, Setup Model associations and enum validations, created rake commands for loading CSV data.

## If PR is linked to a Issue, Add Issue
Database Setup

## PR Checklist
- [x] Added Reviewer
- [x] Followed TDD, And Test are Passing
- [x] Linked Issues for Project Board

### CODE REVIEW

**Review by LancePants97 — APPROVED** [review]

**Review by JCL461437 — APPROVED** [review]
Great work Garrett. Your a monster man.

File: lib/tasks/csv_load.rake [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        Rake::Task["csv_load:customers"].invoke
+        Rake::Task["csv_load:invoice_items"].invoke
+        Rake::Task["csv_load:items"].invoke
+        Rake::Task["csv_load:invoices"].invoke
+        Rake::Task["csv_load:merchants"].invoke
+        Rake::Task["csv_load:transactions"].invoke
+    end
+    
+    # Method to reset_pk_sequence after each CSV load
+    def reset_pk_sequence
```
Comment (austincarrjones) [inline lib/tasks/csv_load.rake:57]: we'll figure this out!

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -63,3 +63,9 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+Shoulda::Matchers.configure do |config|
```
Comment (austincarrjones) [inline spec/rails_helper.rb:66]: sorry I missed this one!

**Review by austincarrjones — APPROVED** [review]
thanks for figuring out the rake task for csv importing!

File: lib/tasks/csv_load.rake (hunk 1 above)
Comment (JCL461437) [reply lib/tasks/csv_load.rake:57]: Yes!

**Pull request merged**